about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Internal CSS -->
+    <link rel="stylesheet" href="./styles/about.css" />
+    <link rel="stylesheet" href="./styles/button.css" />
+    <!-- External CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <title>About</title>
+  </head>
+  <body>
+    <header>
+      <nav class="mobile-nav">
+        <i class="fa-solid fa-bars" id="hamburger"></i>
+      </nav>
+      <nav class="desktop-nav-2">
+        <ul class="flex-list">
+          <li><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
+          <li><a href=""></a><i class="fa-brands fa-twitter"></i></li>
+          <li><a href=""></a>English</li>
+          <li><a href=""></a>My Page</li>
+          <li><a href=""></a>Logout</li>
+        </ul>
+      </nav>
+      <nav class="desktop-nav">
+        <a href="index.html" class="nav-logo" ><img src="./images/school-logo.png" alt=""></a>
+        <ul class="flex-list">
+          <li><a href="about.html">About</a></li>
```

Comment:
- [ ] There should be possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too. When I see there is a link for the about page only but when I try to navigate to the `Home page` There is no link so kindly fix it.